### DISCUSSION TITLE
use yarn to install npm dependency but receive 'warning Workspaces can only be enabled in private projects.'

### DISCUSSION THREAD

**Discussion by hsuyeung** [npm/feedback#680] [CLI]
Example:
1. mkdir test
2. cd test
3. yarn add npm -D 

Result:
![Snipaste_2022-05-10_11-16-37](https://user-images.githubusercontent.com/58101570/167535780-49e63f84-cd5b-4c12-8979-ac6acc0ea658.png)

**Comment by ljharb** [comment]
npm generally shouldn’t be a dev dep; and if you invoked yarn and it doesn’t work as you expect, you need to file an issue with yarn, not with npm.

**Reply by hsuyeung** [reply]
okay，thanks😊
